library/core/src/iter/traits/iterator.rs

```diff
@@ -138,7 +138,7 @@ pub trait Iterator {
     /// A more complex example:
     ///
     /// ```
-    /// // The even numbers from zero to ten.
+    /// // The even numbers from zero to nine.
```

Comment:
```suggestion
    /// // The even numbers in the range of zero to nine.
```

I think it's better to emphasize it's a range. But I also like "The even numbers less than ten." so I'm fine either.